src/server.c

```diff
@@ -4830,6 +4837,7 @@ sds genRedisInfoString(const char *section) {
             "mem_fragmentation_bytes:%zd\r\n"
             "mem_not_counted_for_evict:%zu\r\n"
             "mem_replication_backlog:%zu\r\n"
+            "mem_replication_buffer:%zu\r\n"
             "mem_clients_slaves:%zu\r\n"
```

Comment:
please add a section in the top comment to mention the effect of this PR on interfaces.
i.e. the main effect is an optimization that users won't really notice (it'll just work).
but some info fields changed their purpose, were in the past accumulative and now overlapping.
also mention the new info field.